devel/ansible/roles/dev/tasks/rabbitmq.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Install RabbitMQ packages
+  package:
+      name: "{{ item }}"
+      state: present
+  with_items:
+      - rabbitmq-server
+
+- name: Create RabbitMQ systemd override directory
+  file:
+    path: /etc/systemd/system/rabbitmq-server.service.d/
+    state: directory
+
+- name: Override rabbitmq hostname
```

Comment:
without this, we get an error like

    Feb 27 08:36:05 ipa.example.com rabbitmq-server[13914]: ERROR: epmd error for host ipa: address (cannot connect to host/port)

when trying to start rabbitmq